soroban-sdk-macros/src/lib.rs

```diff
@@ -50,7 +50,12 @@ struct ContractSpecArgs {
 
 #[proc_macro_attribute]
 pub fn contractspecfn(metadata: TokenStream, input: TokenStream) -> TokenStream {
-    let args = parse_macro_input!(metadata as AttributeArgs);
+    let args = match NestedMeta::parse_meta_list(metadata.into()) {
```

Comment:
Why the switch from using syn's `parse_macro_input!` to using darling's `parse_meta_list`?

Reply:
`AttributeArgs` went away -- internal changes in `syn`